appveyor.yml

```diff
@@ -0,0 +1,31 @@
+os:
+  - Visual Studio 2013
+  - Visual Studio 2015
+  - Visual Studio 2017
+shallow_clone: false
+skip_tags: true
+clone_folder: c:\superelastix-source
+configuration:
+  - Release
+platform:
+  - x64
+environment:
+  ITK_URL: https://github.com/InsightSoftwareConsortium/ITK
+  ITK_SOURCE_DIR: c:\ITK-source
+  ITK_BUILD_DIR: c:\ITK-build
+  ELASTIX_URL: https://github.com/SuperElastix/elastix
+  ELASTIX_SOURCE_DIR: c:\elastix-source
+  ELASTIX_BUILD_DIR: c:\elastix-build
+  SUPERELASTIX_SOURCE_DIR: c:\superelastix-source\SuperBuild
+  SUPERELASTIX_BUILD_DIR: c:\superelastix-build
+cache:
+  - '%ITK_SOURCE_DIR%'
+  - '%ITK_BUILD_DIR%'
+  - '%ELASTIX_SOURCE_DIR%'
+  - '%ELASTIX_BUILD_DIR%'
+build_script:
+  - cmd: if not exist %ITK_SOURCE_DIR% git clone %ITK_URL% %ITK_SOURCE_DIR% && cd %ITK_SOURCE_DIR% && git checkout v4.12.2
+  - cmd: if not exist %ITK_BUILD_DIR% mkdir -p %ITK_BUILD_DIR% && cd %ITK_BUILD_DIR% && cmake -G "Visual Studio 12 2013 Win64" -DBUILD_EXAMPLES=OFF -DBUILD_TESTING:BOOL=OFF -DITK_BUILD_DEFAULT_MODULES:BOOL=OFF -DITKGroup_IO:BOOL=ON -DModule_ITKDistanceMap:BOOL=ON -DModule_ITKImageFusion:BOOL=ON -DModule_ITKLabelMap:BOOL:=ON -DModule_ITKMathematicalMorphology:BOOL=ON -DModule_ITKOptimizers:BOOL=ON -DModule_ITKOptimizersv4:BOOL=ON -DModule_ITKRegistrationCommon:BOOL=ON -DModule_ITKVideoIO:BOOL=ON-DCMAKE_BUILD_TYPE=Release %ITK_SOURCE_DIR% && cmake --build . --config Release -- /m
```

Comment:
Is it correct that -g "Visual Studio 12 2013 win64" is hard coded here? On top you define other versions of Visual Studio too.

Reply:
It has been fixed in other branches.